tutorials-in-development/Staging_Area/02_ecocomDP_workflow_with_NEON_algae.Rmd

```diff
@@ -201,7 +201,7 @@ alg_observations_with_taxa %>% head()
 
 We can also make a quick plot to see which taxon rank (i.e., what level of taxonomic specificity was achieved by the expert taxonomist) is most common:
 
-```{r plot-taxon-rank}
+```{r plot-taxon-rank, fig.cap= "Bar plot showing counts of algae taxonomic ranks at the Arikaree River site."}
```

Comment:
"Bar plot showing the frequency of each taxonomic rank observed in algae count data from the Arikaree River site."

Reply:
Thanks for reviewing, @sokole . Updated.